reconcile/utils/jump_host.py

```diff
@@ -41,13 +41,29 @@ def cleanup(self):
 
 class JumpHostSSH(JumpHostBase):
     bastion_tunnel = {}
+    local_ports = []
     tunnel_lock = threading.Lock()
 
     def __init__(self, jh, settings=None):
         JumpHostBase.__init__(self, jh, settings=settings)
 
         self.known_hosts = self.get_known_hosts(jh)
         self.init_known_hosts_file()
+        self.local_port = self.get_random_port() \
```

Comment:
If you have a separate getter for this variable, should this be considered internal?  ex. `self._local_port` and `self._remote_port`?  Alternatively is there any reason not to allow direct access and you can always use properties if needed later?